### ISSUE TITLE
Uncaught TypeError: Cannot read property 'commands' of null

### ISSUE THREAD

**Issue by doitajin** [Codiad/Codiad#976]
Uncaught TypeError: Cannot read property 'commands' of null
    at init.js:43
    at Object.publish (amplify.min.js:10)
    at Object.fn [as success] (init.js:91)
    at o (jquery.min.js:2)
    at Object.fireWith [as resolveWith] (jquery.min.js:2)
    at w (jquery.min.js:4)
    at XMLHttpRequest.d (jquery.min.js:4)

**Comment by seanvree** [comment]
Yeah, I just started to get this as well.  I can't load the UI at all now.  Was working for weeks, then just stopped. Totally crashes the browser (Chrome)

OS:  Windows 10E x64 //  
Webserver:  IIS 10 
PHP:  7.1.7 x64

![image](https://user-images.githubusercontent.com/8906637/29465092-dc0786c8-83ec-11e7-9c57-98824e81da8b.png)



@doitajin  did you ever figure this out?
